templates/article.jade

```diff
@@ -0,0 +1,25 @@
+
+extends layout
+
+block append vars
+  - bodyclass = 'article-detail'
+
+block prepend title
+  | #{ page.title + " - "}
```

Comment:
Inconsistent spacing between `#{` and other instances. Prob go for `#{page.title + " - "}`